.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 
 install:
   - sudo apt-get install rethinkdb
-  - pip install -e .[test]
+  - pip install -e .[test] --process-dependency-links
```

Comment:
1. Push latest BDB-common to pip
2. Remove this label
3. Update version in setup.py